addons/website_sale/models/delivery_carrier.py

```diff
@@ -12,3 +16,53 @@ class DeliveryCarrier(models.Model):
         related='product_id.description_sale',
         readonly=False,
     )
+    has_estimated_delivery = fields.Boolean(
+        string='Estimated Delivery',
+        help="Display estimated date to your customer. Editable if range is defined."
+    )
+    estimated_delivery_start = fields.Integer()
+    estimated_delivery_end = fields.Integer()
+    opening_hours = fields.Many2one(
+        string="Opening Hours", comodel_name='resource.calendar', check_company=True
+    )
```

Comment:
nit: the string is redundant as it will be the default (cf. internal tooling discord).

```suggestion
    opening_hours = fields.Many2one(comodel_name='resource.calendar', check_company=True)
```